### DISCUSSION TITLE
What is the reason I was restricted by github?

### DISCUSSION THREAD

**Discussion by KevynHuang** [community/community#32368] [Other Feature Feedback, Questions, & Ideas]
<img width="693" alt="image" src="https://user-images.githubusercontent.com/93977721/189259406-4318a566-6ea9-43be-aba8-155efcb1393d.png">

These days ,I have seen these  restriction a few times, I don't think I did anything wrong, is there any guys who could make a explanations for me ？thank you !

**Comment by ahhTou** [comment]
他妈的，还有这事，揍他丫的

**Reply by kcoderk** [reply]
bulls\*\*t g\*\*hub

**Comment by airtower-luna** [comment]
"Rate limit" implies too many requests in quick succession. I can't say why that may be. Are you possibly on a network where a lot of users who use GitHub share an external IP address?

**Reply by KevynHuang** [reply]
Ya,but for a company,It's normalm.right?

**Reply by airtower-luna** [reply]
Depends on the size of the company. :sweat_smile: With a few people, yes. I don't know where it gets "too much" for their rate limits, I'm just saying what the message indicates.

**Comment by skadomsky** [comment]
Same situation using GitHub in China. I guess it's mostly GitHub's measure to prevent spam requests from Chinese IP.
Just be calm, drink a cup of tea, and use vpn amap.
